lib/generators/gql/gql_generator_base.rb

```diff
@@ -37,5 +37,32 @@ def map_model_types(model_name)
           .map { |col| {name: col.name, gql_type: type_map[col.type]} }
       end
     end
+
+    def root_directory(namespace)
+      "app/graphql/#{namespace.underscore}"
+    end
+
+    def wrap_in_namespace(namespace)
+      namespace = namespace.split('::')
+      namespace.shift if namespace[0].empty?
+
+      code = namespace.each_with_index.map { |name, i| "  " * i + "module #{name}" }.join("\n")
+      code << "\n" << yield(namespace.size) << "\n"
+      code << namespace.size.downto(0).map { |i| "  " * i  + "end" }.join("\n")
```

Comment:
FIXME: should be `downto(1)` and add a `"end"` to yielded code.

Reply:
@sshaw was this resolved?

Reply:
No.